### PR TITLE
Add drag and drop

### PR DESCRIPTION
**Pull request by vinayak-4**

### CODE REVIEW

File: src/pages/Profile/index.jsx [hunk 1]
```diff
@@ -29,12 +37,23 @@ export default function Profile() {
       }
     };
   }
-  function isAvailableInTheList(list) {
-    return function (id) {
-      return list.find((game) => game.id === id);
-    };
+
+  function dragStartHandler(list, setList, data, ev) {
+    setDataTransfer({ from: [list, setList], data: { ...data } });
```
Comment (Delwalt) [inline src/pages/Profile/index.jsx]: this function can be further simplified, you don't need to store `list` and `setList` variable, all you need to do is store data of  card you are dragging and from its status you can know from which section you are dragging it, for example if its status is `want to play` then you know this card belongs to `want to play` list.

File: src/pages/Profile/index.jsx [hunk 2]
```diff
@@ -29,12 +37,23 @@ export default function Profile() {
...
+    ev.dataTransfer.dropEffect = "link";
   }
 
+  function dropHandler(list, setList, event) {
+    event.preventDefault();
```
Comment (Delwalt) [inline src/pages/Profile/index.jsx:68]: The same goes here, you can pass status as an argument here, and based on the drop status and drag card status you know where to move the card.

**Review by Delwalt — CHANGES_REQUESTED** [review]

File: src/pages/Profile/index.jsx (hunk 1 above)
Comment (vinayak-4) [reply src/pages/Profile/index.jsx]: then we need to add if conditions for three status (want to play, playing, played) in function?

Comment (vinayak-4) [reply src/pages/Profile/index.jsx]: in this we just need to pass the list which needs to be updated

Comment (Delwalt) [reply src/pages/Profile/index.jsx]: that is ok to handle multiple condition, but that way it is  much more clear

Comment (vinayak-4) [reply src/pages/Profile/index.jsx]: Fixed

File: src/pages/Profile/index.jsx (hunk 2 above)
Comment (vinayak-4) [reply src/pages/Profile/index.jsx:68]: Fixed

**Review by Delwalt — APPROVED** [review]

**Pull request merged**